hw/ip/lc_ctrl/data/lc_ctrl.hjson

```diff
@@ -644,5 +650,29 @@
         }
       ]
     }
+
+    ///////////////
+    // Device ID //
+    ///////////////
+    { multireg: {
```

Comment:
@mcy FYI, this is a new CSR that exposes a non-secret, unique device ID. The device ID is part of the `HW_CFG` partition in OTP, and becomes visible here once provisioned.